### PR TITLE
Add : OG Image generation for all social media

### LINKED ISSUE #31: Format all files with Prettier

### LINKED ISSUE #34: Open Graph (OG) Image generation
Create OG images for various social media platforms [Learn more here](https://vercel.com/docs/functions/edge-functions/og-image-generation/og-image-examples)

[Video on OG images 
](https://youtu.be/6rYzM_oWi7A?si=2gDnsKQl00WjCsok)

### PR DESCRIPTION
**Pull request by PriyansuMaurya**
## Fixes Issue
Closes #34

## Screenshots

New OG image

203shots_so

### CODE REVIEW

File: pages/index.jsx
Comment (urdadx) [inline pages/index.jsx]: The description of the project in the metadata does not hold for Librelinks. I think you have mistaken it for another project.

**Review by urdadx — CHANGES_REQUESTED** [review]
Change the description of the metatag

File: pages/index.jsx
Comment (PriyansuMaurya) [reply pages/index.jsx]: Yes, I bad. Wait, I will fix it.

**Comment by PriyansuMaurya** [comment]
For the mean time Authorization it to deploy, to check if other things are working fine.

**Review by urdadx — APPROVED** [review]

**Comment by urdadx** [comment]
Works fine on WhatsApp. Does not work for Twitter

image

**Comment by PriyansuMaurya** [comment]
You should have provided approval for deployment at this specific location, reviewed the changes using the deployment link, and merged it once everything was functioning correctly.

In any case, I will now make changes here. Your role is to approve deployment here, verify if it work correctly with the link generated by vercel, and then merge it in final step.

**Comment by PriyansuMaurya** [comment]
It also work fine on 

Signal

image

Discord 

image

also on pinterest.

**Pull request merged**
